### ISSUE TITLE
Process 'command '/usr/local/bin/docker-compose'' finished with exit value 125 (state: FAILED)

### ISSUE THREAD

**Issue by gkns** [avast/gradle-docker-compose-plugin#357]
I can manually run the docker-compose from command-line as: 
`docker-compose -f docker-compose/integ-test.yml up`

However, when I try the same with the below DSL:

```
apply plugin: "com.avast.gradle.docker-compose"
...
dockerCompose {
    // settings as usual
    integTest {
        useComposeFiles = ["$rootDir/docker-compose/integ-test.yml"]
        projectName = "cpath_test"
    }
}
```

I get this error:
-------
2022-05-12T20:46:00.159+0530 [INFO] [org.gradle.process.internal.DefaultExecHandle] Starting process 'command '/usr/local/bin/docker-compose''. Working directory: /Users/naranathusig/git-repos/cpn-auth/auth-service Command: /usr/local/bin/docker-compose -f /Users/naranathusig/git-repos/cpn-auth/docker-compose/integ-test.yml -p cpath_test version --short
2022-05-12T20:46:00.159+0530 [DEBUG] [org.gradle.process.internal.DefaultExecHandle] Changing state to: STARTING
2022-05-12T20:46:00.159+0530 [DEBUG] [org.gradle.process.internal.DefaultExecHandle] Waiting until process started: command '/usr/local/bin/docker-compose'.
2022-05-12T20:46:00.165+0530 [DEBUG] [org.gradle.process.internal.DefaultExecHandle] Changing state to: STARTED
2022-05-12T20:46:00.165+0530 [DEBUG] [org.gradle.process.internal.ExecHandleRunner] waiting until streams are handled...
2022-05-12T20:46:00.165+0530 [INFO] [org.gradle.process.internal.DefaultExecHandle] Successfully started process 'command '/usr/local/bin/docker-compose''
2022-05-12T20:46:00.360+0530 [[31mERROR[0m] [system.err] unknown shorthand flag: 'f' in -f
2022-05-12T20:46:00.361+0530 [[31mERROR[0m] [system.err] See 'docker --help'.
2022-05-12T20:46:00.361+0530 [[31mERROR[0m] [system.err] 
2022-05-12T20:46:00.361+0530 [[31mERROR[0m] [system.err] Usage:  docker [OPTIONS] COMMAND
2022-05-12T20:46:00.361+0530 [[31mERROR[0m] [system.err] 
2022-05-12T20:46:00.361+0530 [[31mERROR[0m] [system.err] A self-sufficient runtime for containers
...
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err]   update      Update configuration of one or more containers
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err]   version     Show the Docker version information
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err]   wait        Block until one or more containers stop, then print their exit codes
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err] 
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err] Run 'docker COMMAND --help' for more information on a command.
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err] 
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err] [1mTo get more help with docker, check out our guides at https://docs.docker.com/go/guides/[0m
2022-05-12T20:46:00.364+0530 [[31mERROR[0m] [system.err] 
2022-05-12T20:46:00.364+0530 [DEBUG] [org.gradle.process.internal.DefaultExecHandle] Changing state to: FAILED
2022-05-12T20:46:00.364+0530 [DEBUG] [org.gradle.process.internal.DefaultExecHandle] Process 'command '/usr/local/bin/docker-compose'' finished with exit value 125 (state: FAILED)

The issue seems to be in the order of command-line arguments supplied to the docker-compose.
Can one of you help to get past this issue?

**Comment by augi** [comment]
Hi, as you can see in the log, the plugin first reads the version of Docker Compose, using the following command:
`/usr/local/bin/docker-compose -f /Users/naranathusig/git-repos/cpn-auth/docker-compose/integ-test.yml -p cpath_test version --short`

It looks like this command crashes on your machine 🤔 

As stated in the documentation, `-f` is a valid flag: https://docs.docker.com/compose/reference/

**Comment by gkns** [comment]
When I run this command manually, I see the same error related to the command line parameters, If I tweak the command to
`/usr/local/bin/docker-compose -f /Users/naranathusig/git-repos/cpn-auth/docker-compose/integ-test.yml up`
It works!

**Comment by gkns** [comment]
What I ended up doing is using a couple of bash aliases for docker-compose up and removing the plugin from builds. This is bad since it involves manual steps. But for local development, this will work in the short term.

Let me know if you have any debug build/branch that I can try.
Thanks

**Comment by gkns** [comment]
> Addressed in [0.16.5](https://github.com/avast/gradle-docker-compose-plugin/releases/tag/0.16.5), hope it will help 🤞

Thank you! I can confirm that it works on my dev machine.
I will circle back on the rate occasion that it fails on another platform our CI is running (cent OS)
